src/libmongoc/src/mongoc/mongoc-bulkwrite.c

```diff
@@ -1438,12 +1445,36 @@ _bulkwritereturn_apply_result (mongoc_bulkwritereturn_t *self,
    return true;
 }
 
+void
```

Comment:
Just removed the `MONGOC_EXPORT` here per https://github.com/mongodb/mongo-c-driver/pull/1806#discussion_r1873582510.